### PR TITLE
Fixed comment position to be the offset from the beginning of the hun…

### PR DESCRIPTION
**Pull request by riot-mraviv**
…k instead of the offset from beginning of the file diff. Fixed the comments count to count successfully posted comments. Added some user friendly output regarding unposted comments.

### CODE REVIEW

File: inlineplz/interfaces/github.py [hunk 1]
```diff
@@ -98,38 +98,46 @@ def post_messages(self, messages, max_comments):
...
                 self.pull_request.create_review_comment(
                     self.format_message(msg),
                     self.last_sha,
                     msg.path,
                     msg_position
                 )
-            except github3.GitHubError:
-                pass
+            except github3.GitHubError as err:
+                print("Posting failed: " + str(err))
```
Comment (guykisel) [inline inlineplz/interfaces/github.py]: you could use `.format()` here to avoid explicitly casting to a string. it's fine as-is though.

**Comment by guykisel** [comment]
looks good to me! will merge and release when the tests pass

**Comment by riot-mraviv** [comment]
:)

**Comment by guykisel** [comment]
failure is due to a rate limit issue, not a legit issue.

**Pull request merged**